dsls/sql/src/main/java/org/apache/beam/dsls/sql/schema/BeamSqlRowCoder.java

```diff
@@ -49,6 +50,8 @@
   private static final DoubleCoder doubleCoder = DoubleCoder.of();
   private static final InstantCoder instantCoder = InstantCoder.of();
   private static final BigDecimalCoder bigDecimalCoder = BigDecimalCoder.of();
+  private static final SerializableCoder<Boolean> booleanCoder =
```

Comment:
I suspect a `ByteCoder` is better than `SerializableCoder` in terms of performance and deterministcity?